src/confcom/azext_confcom/_help.py

```diff
@@ -93,6 +93,18 @@
           type: boolean
           short-summary: 'When enabled, the generated policy will not contain the ID field. This will keep the policy from being tied to a specific image name and tag.'
 
+        - name: --include-fragments -f
+          type: boolean
+          short-summary: 'When enabled, image-attached fragments will be pulled and used for generating the policy or the path specified by --fragments-json will be used'
```

Comment:
I think it can be heterogenous within a single container group. If there's both a fragment in the remote we can pull, and it's defined in `--fragments-json` we'll compare against both of them but eventually only one will be used depending on which covers the container fully

Reply:
not sure if there's merging logic here. what do you think of 

> When enabled, image-attached fragments will be pulled for generating the policy. If --fragments-json is also provided, both sources will be compared, with the fragment providing the most complete coverage for the container group used. Partial fragments will be merged if possible. If no --fragments-json is specified, only image-attached fragments will be used.

Reply:
Will add this wording when other options for including fragments are enabled